style.css

```diff
@@ -0,0 +1,283 @@
+@media (min-width: 375px) {
+  body {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0 !important;
+    height: 100vh;
+  }
+
```

Comment:
- Kindly add the background image missing on your mobile version

![image](https://user-images.githubusercontent.com/93630700/227519138-af807d75-a461-432b-a0aa-8734a7daf13a.png)